packages/bus-mongodb/src/mongodb-persistence.ts

```diff
@@ -10,7 +10,7 @@ import { Message, MessageAttributes } from '@node-ts/bus-messages'
 import { Db, MongoClient } from 'mongodb'
 import { MongodbConfiguration } from './mongodb-configuration'
 import { WorkflowStateNotFound } from './error'
-import mapKeys from 'lodash.mapkeys'
+import _ from 'lodash'
```

Comment:
`lodash` isn't part of the dependencies, and it's quite a large package to pull in just for `mapKeys`.

Could this be replaced with a quick implementation instead? something like:

```suggestion

const mapKeys = (obj: any, fn: (key: string, value: any) => string) => {
  return Object.keys(obj).reduce((acc, oldKey) => {
    const newKey = fn(oldKey, obj[oldKey])
    acc[newKey] = obj[oldKey]
    return acc
  }, {} as Record<string, unknown>)
}
```

Reply:
Good point, I will apply it and push the solution again. 
Thanks

Reply:
@adenhertog I have applied the fix. Check again.

Reply:
looks great, nice one